meedan/check-web
# Fix/CV2-4038: Redirect user when attempting to join a shared feed that user already belongs to

## Description
- when the user clicks on the e-mail to accept the invitation, if another team member already accepts the invitation, take the user straight to the shared feed.
- when the user has two teams or more the user is redirected directly to the check invitation screen, where the user can choose which team they will accept the invitation, the user should be able to see which teams already are part of the shared feeds. 
Reference: CV2-4038

## Type of change


## How has this been tested?

Scenario 1:

Steps to Test:
User A and User B are in Workspace A.
User C invites User A and User B to join a shared feed.
User A and User B navigate to the page to accept the invitation.
User A accepts the invitation.
Verify that no error message is displayed and that the invitation is successfully accepted.
User B attempts to accept the invitation shortly after User A.
Verify that no error message related to a unique key constraint conflict from the database appears.
Verify that User B is redirected to the edit feed page

Scenario 2:

Steps to Test:
User A and User B are in Workspace A.
User C invites User A and User B to join a shared feed.
User A accepts the invitation.
Verify that no error message is displayed and that the invitation is successfully accepted.
User B clicks on the invitation shortly after User A accept the invitation.
Verify that User B is NOT be taken to the invitation page, they should be taken straight to the shared feed.

Scenario 3:
Steps to Test:
User B receives the invitation and goes to the URL http://localhost:3333/check/feed/feedID/invitation to choose the team 
Verify that user B is able to see which teams already are part of the shared feeds


## Checklist
